Source/Client/Sound/SoundSystem.cs

```diff
@@ -275,6 +275,7 @@ public static void CreateSound(string filename, string fullfilename)
 
             // Add to collection
             sounds.Add(filename, s);
+            return s;
```

Comment:
Hmm, it seems like if we return it from here, and then if jukebox wants to use the same sound once again, it will throw as the sound was already loaded.

So we should manually create Sound inside Track

Reply:
Or maybe instead of throwing here, we should just return existing sound? If so, Track's Dispose method should not call Dispose on owned ISound